src/services/schedule.ts

```diff
@@ -413,6 +413,13 @@ async function matchStrategyPrep(
   return { transaction, isStrategyTx, isLastPrepForNonce: strategyPrep.isLastForNonce };
 }
 
+function SendEmailForNonStrategyTx(scheduled: IScheduled) {
+  const isStrategyTx = scheduled.strategyPrepId;
```

Comment:
And this was double negated (`!!`) to convert it into a boolean and now it's not for some reason. This might not matter in this case but it's good to use proper types always